schemas/forum.sql

```diff
@@ -396,6 +396,7 @@ CREATE TABLE `users` (
   `digest` char(1) DEFAULT 'Y',
   `timezone` varchar(48) DEFAULT NULL,
   `moderator` char(1) DEFAULT 'N',
+  `admin` char(1) DEFAULT 'N',  
```

Comment:
could you please provide a separated sql file for manual migration? see `schemas` dir